params.yaml

```
@@ -0,0 +1,41 @@
+graph-defaults:
+  levels: 4
+  hierarchical: true
+  name: hierarchical
```

```suggestion
  name: &graph_name hierarchical_4lvl
```
To make it more clear that this is the name of the graph, not the type we could write it like this. Also use the same value when training the model.